[ews-app] Add basic github support

aj062 opened the pull request:
#### d6484ed66c865d78157f2d560ef276ada4145dde
<pre>
[ews-app] Add basic github support
<a href="https://bugs.webkit.org/show_bug.cgi?id=242940">https://bugs.webkit.org/show_bug.cgi?id=242940</a>

Reviewed by Jonathan Bedard.

Mostly copied from GitHub class in <a href="https://github.com/WebKit/WebKit/blob/main/Tools/CISupport/ews-build/steps.py">https://github.com/WebKit/WebKit/blob/main/Tools/CISupport/ews-build/steps.py</a>
* Tools/CISupport/ews-app/ews/common/github.py: Added.

Canonical link: <a href="https://commits.webkit.org/252645@main">https://commits.webkit.org/252645@main</a>
</pre>

JonWBedard commented on Tools/CISupport/ews-app/ews/common/github.py:
```diff
@@ -0,0 +1,134 @@
+            return cls._cache[prefix]
+
+        try:
+            passwords = json.load(open('passwords.json'))
```
Does ews-app use `passwords.json` like ews-build does?

aj062 replied:
yes

aj062 committed:
[ews-app] Add basic github support
https://bugs.webkit.org/show_bug.cgi?id=242940

Reviewed by Jonathan Bedard.

Mostly copied from GitHub class in https://github.com/WebKit/WebKit/blob/main/Tools/CISupport/ews-build/steps.py
* Tools/CISupport/ews-app/ews/common/github.py: Added.

Canonical link: https://commits.webkit.org/252645@main
